### DISCUSSION TITLE
`jQuery@3.6.4` error on update

### DISCUSSION THREAD

**Discussion by UlrichHP** [jquery/jquery#5232] [Q&A] [answered]
Hi,
I have a Sylius/Symfony project and when i tried to update jQuery from `3.6.3` to `3.6.4`, it is no longer working.

I have several errors in the console and when i try to `console.log(jQuery)`, it returns what you can see in the screenshot below.
Is anyone else having this problem ?

Thanks

![image](https://user-images.githubusercontent.com/27560999/227894865-39d5ffb8-2dde-44ad-ba04-bdc8db730a0f.png)

**Comment by mgol** [comment, marked as answer]
The error you're getting comes from a jQuery plugin so we cannot do much here. I'm moving it to discussions since it's not an actionable bug report.

I don't know why it would break like that on updating jQuery from 3.6.3 to 3.6.4 but it's highly likely it's only related to how you load jQuery. Are you doing it in exactly the same way as you did with jQuery 3.6.3? Did you change anything?

**Reply by UlrichHP** [reply]
Thanks for your answer.

No, nothing has changed between the 2 versions. The only thing i did was updating jquery from 3.6.3 to 3.6.4 with yarn berry (v3.5.0) upgrade-interactive command

**Reply by mgol** [reply]
The differences between jQuery 3.6.3 & 3.6.4 are really small:
[jquery/jquery@`3.6.3..3.6.4`#diff-f95249cff4](https://github.com/jquery/jquery/compare/3.6.3..3.6.4#diff-f95249cff4afb838670287f45229cd581995f69ca5a5a47d2dcbc22b9ce6fe9e)
so it's highly unlikely it causes some plugin code to not be available.

What's more likely is that this upgrade didn't fully replace jQuery for you or that you have two copies. I'd review the generated changes via Git diff carefully.

**Reply by UlrichHP** [reply]
Yes that's why i don't understand why it is not working anymore...

I will try to see if there are two copies of jQuery like you said.
Thanks for your help

**Reply by UlrichHP** [reply]
Just to let you know, it seems there is a problem with Sylius.
I installed their SyliusStandard project skeleton and only updated jQuery to v3.6.4 and i have the same errors.
I will try to contact them.

Thanks again for your help
